scripts/halium

```diff
@@ -74,6 +74,11 @@ identify_android_image() {
 	[ -f /halium-system/var/lib/lxc/android/system.img ] && ANDROID_IMAGE_MODE="system"
 	[ -f /halium-system/var/lib/lxc/android/android-rootfs.img ] && ANDROID_IMAGE_MODE="rootfs"
 	[ -z $ANDROID_IMAGE_MODE ] && ANDROID_IMAGE_MODE="unknown"
+
+	if [ "$ANDROID_IMAGE_MODE" = "unknown" ]; then
+		halium_panic "Could not identify Android system image!"
+	fi
```

Comment:
Ok I´ll remove it. The problem remains, why continue trying to boot if something essential is missing? If the system.img is missing then a bunch of errors is written out, and still the thing tries to start LXC it seems. I wanted to have a clear stop condition somewhere so that people can see, okay at the very end of the log: This is the problem.